plugins/node/instrumentation-fs/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@opentelemetry/instrumentation-fs",
+  "version": "0.27.0",
+  "description": "OpenTelemetry automatic instrumentation package for fs",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "test": "ts-mocha -p tsconfig.json 'test/**/*.test.ts'",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version && lerna run version --scope @opentelemetry/instrumentation-fs --include-dependencies",
+    "prewatch": "npm run precompile",
+    "prepare": "npm run compile",
+    "version:update": "node ../../../scripts/version-update.js",
+    "compile": "npm run version:update && tsc -p ."
+  },
+  "keywords": [
+    "fs",
+    "instrumentation",
+    "nodejs",
+    "opentelemetry",
+    "profiling",
+    "tracing"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": "^1.0.2"
+  },
+  "devDependencies": {
+    "@opentelemetry/api": "1.0.2",
```

Comment:
may want to update to latest deps

Reply:
If the user uses an old 1.x.x version, this will force him to upgrade to a higher api version in order to use the instrumentation, when in practice older versions will work fine as well, right?
I find myself sometimes wondering if the version should be the minimum supported one (1.0.0), or the latest 🤔

Reply:
It makes sense for the implementors/libs to specify the smallest version supported to maximize compatibility across libs.

In that sense, having a core package expose interfaces to implement messes with the semver expectations.

But the I don't see a reason to hold back updating the dev deps.

Reply:
As far as I know renovate bot will create a PR to pin it in case you would use e.g. `^1.0.2`. But on the other hand it seems renovate PRs are mostly ignored anyway meanwhile otherwise we would have 1.0.4 at other places.

Reply:
> But the I don't see a reason to hold back updating the dev deps.

I think CI enforces that the version of `@opentelemetry/api` in peerDependencies matches the version in devDependencies

Reply:
That's true. I'll address that separately in #936 and update all the packages after that's merged. For now. I'll comply with the restrictions peer-check currently enforces.

> As far as I know renovate bot will create a PR to pin it in case you would use e.g. ^1.0.2. But on the other hand it seems renovate PRs are mostly ignored anyway meanwhile otherwise we would have 1.0.4 at other places.

I agree we could be better with updating the minor versions of packages. It's currently problematic because renovate bot bundles them all together and some of them fail tests - #618

Probably not a huge deal, but requires a significant contribution to resolve.